src/components/ButtonStripe.tsx

```diff
@@ -19,7 +19,7 @@ export default function ButtonStripe(props: {
         { price: props.price_id, quantity: 1 },
       ],
       mode: "subscription",
-      clientReferenceId: props.donorname + sep + props.forumname,
+      clientReferenceId: (props.donorname || "") + sep + (props.forumname || ""),
```

Comment:
While this is fine, you might want to use `??` in the future, as there is a slight difference.

Reply:
Thanks for the hint, I don't have that much JS experience. What would the difference be in this case?

Reply:
Please see the example at https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing